controllers/usersController.ts

```diff
@@ -0,0 +1,122 @@
+import * as validate from './utils/validate.js';
+import bcrypt from 'bcrypt';
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient()
+
+export const getAllUsers = (req: any, res: any) => {
+    res.send("hit get users route");
+}
+
+
+export const registerUser = async (req: any, res: any) => {
+
+    // I want to make this to a middleware? Or like a reusable function so that the isValid checking is not repeated and the reqBody properties would be available already as variables.
+    const neededKeys = ['email', 'username', 'firstName', 'lastName', 'password'];
+
+    const { reqBody, isValid } = validate.bodyVsKeys(req.body, neededKeys);

```

Comment:
reading `validate.bodyVsKeys` doesnt make sense, can we rename it to something more understandable?

Reply:
Got it. Renaming now